tests/RobotFramework/tests/plugin_apt/install_overwrite_config_files.robot

```diff
@@ -0,0 +1,52 @@
+*** Settings ***
+Resource            ../../resources/common.resource
+Library             ThinEdgeIO
+Library    Cumulocity
+
+Suite Setup         Custom Setup
+Suite Teardown      Get Logs
+
+Test Tags           theme:software    theme:plugins
+
+
+*** Test Cases ***
+Install packages without overwriting config files
+    Execute Command    tedge config set apt.dpk.options.config keepold
+    # install package v1
+    Execute Command   /etc/tedge/sm-plugins/apt install /setup/sampledeb_1.0.0_all.deb  
+    # Modify the config file
+    Execute Command    echo "Updated the config" | sudo tee -a /etc/sampledeb.cfg
+
+    # install package v2
+    Execute Command   /etc/tedge/sm-plugins/apt install /setup/sampledeb_2.0.0_all.deb  
+    # Assert to make sure installation of newer version did not update the config file
+    ${output}=    Execute Command    cat /etc/sampledeb.cfg
+    Should Contain    ${output}    conf 1.0
+    Should Contain    ${output}    Updated the config
+    ThinEdgeIO.Remove Package Using APT    sampledeb
+    Execute Command   rm /etc/sampledeb.cfg
+
+
+Install packages overwrite config files
+    Execute Command    tedge config set apt.dpk.options.config keepnew
+    # install package v1
+    Execute Command   /etc/tedge/sm-plugins/apt install /setup/sampledeb_1.0.0_all.deb  
```

Comment:
I'm actually surprised calling the `apt` sm plugin works like this.

I think we should be calling it like how it would be done in a real situation, e.g.

```
/etc/tedge/sm-plugins/apt install sampledeb --file /setup/sampledeb_1.0.0_all.deb
```

Reply:
Yes, it works. But now I fixed it, as suggested above.